conf/application.conf

```diff
@@ -180,7 +180,7 @@ tracking-consent-frontend {
 }
 
 features {
-  agent-on-behalf-of-trader: true
+  agent-on-behalf-of-trader: false
```

Comment:
Just a note, there's a ticket next sprint to literally do this one liner haha, for now that's fine